### ISSUE TITLE
Index pages that have children aren't reachable in the nav menu

### ISSUE THREAD

**Issue by warpfork** [warptools/warpforge-site#13]
Pages with children aren't reachable in the Nav menu because they become menu-unfold buttons rather than regular links.

We could deal with this in two ways: either we tweak the nav menu so there's both links and unfold buttons for pages that are parents; or, we could just... structure our content to not put information in those parent pages.

I'm actually fine with either (and at least for the moment, shifting towards the later, so I can just keep writing content today without blocking).  But it would give me a least a little pause to skip those parent pages entirely, because for any page `/a/b/c`, I'd like `/a/b/` to have at least some skeletal content in case someone navigates there by sheer manipulation of URLs.  (I dunno how common it is, but I for one _am that guy_ who manually munges URL chunks and expects parent paths to exist, heh.)

**Comment by Storken** [comment]
Yeah, this is a tricky one 🤔 we currently only have the index page of a parent because the way we've built the nav requires it, but that doesn't mean we can for example add sub-links to the children pages of the parent.

**Comment by Storken** [comment]
It would be cool to add the links to the children-pages in some kind of automatic manner, i'll look into this!
